docs/tutorial/part-two/index.md

```diff
@@ -486,16 +297,33 @@ export default () => (
 )
 ```
 
+> Tip: Generally, if you use a component in multiple places on a site, it should be in its own module file in the `components` directory. But, if it's used only in one file, create it inline.
+
 The finished page should now look like:
 
 ![css-modules-final](css-modules-final.png)
 
+### CSS-in-JS
+
+CSS-in-JS is a component-oriented styling approach. Most generally, it is a pattern where [CSS is composed inline using JavaScript](https://reactjs.org/docs/faq-styling.html#what-is-css-in-js).
+
+#### Using CSS-in-JS with Gatsby
+
+There are many different CSS-in-JS libraries, and many of them have Gatsby plugins already. We won't cover an example of CSS-in-JS in this initial tutorial, but we encourage you to [explore](/docs/styling/) what the ecosystem has to offer. There are mini-tutorials for two libraries in particular, [Glamor](/docs/glamor/) and [Styled Components](/docs/styled-components/).
```

Comment:
We need an Emotion tutorial. ☹️ 👩‍🎤

Reply:
I was thinking this as I was doing this rewrite. I'm on a roll, I might go ahead and do it.

That reminds me, something on the todo i need to add is some tweaks to some of the related style docs.